frontend/src/components/samples/SamplesListContent.js

```diff
@@ -202,6 +205,7 @@ const SamplesListContent = ({
     <AppPageHeader title="Samples" extra={[
       <AddButton key='add' url="/samples/add" />,
       ...actionsToButtonList("/samples", actions),
+      ...prefillTemplatesToButtonDropdown(withToken(token, api.samples.prefill.request), prefills),
```

Comment:
no need for ... before prefillTemplatesToButtonDropdown since you are not returning a list, you are returning a single menu dropdown. actionsToButtonList needs ... because it returns a list of buttons